pom.xml

```diff
@@ -148,6 +148,8 @@ under the License.
 		<hivemetastore.hadoop.version>2.7.5</hivemetastore.hadoop.version>
 		<jamicmp.referenceVersion>1.8.0</jamicmp.referenceVersion>
 		<japicmp.outputDir>tools/japicmp-output</japicmp.outputDir>
+		<scheduler.type>legacy</scheduler.type>
```

Comment:
Is it necessary to set a default scheduler in the build configuration again? We already have a default set in Java code:

```
final String schedulerType = System.getProperty(SCHEDULER_TYPE_KEY, JobManagerOptions.SCHEDULER.defaultValue());
```

Reply:
You are right. I think we can remove it.